imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/CCITTFaxDecoderStream.java

```diff
@@ -366,17 +382,7 @@ private int decodeRun(final Tree tree) throws IOException {
     }
 
     private void resetBuffer() throws IOException {
-        for (int i = 0; i < decodedRow.length; i++) {
-            decodedRow[i] = 0;
-        }
-
-        while (true) {
-            if (bufferPos == -1) {
-                return;
-            }
-
-            readBit();
-        }
+        bufferPos = -1;
```

Comment:
Would it be possible to create an "artificial" sample stream for a test case? As usual, I'm not worried that the current implementation is correct. It's just that I can change it without breaking any tests. This is bound to happen sooner or later as part of some refactoring...